Data/Models/ProductTables/UpdateProductInput.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Models.ProductTables
+{
+    public class UpdateProductInput
```

Comment:
You should not put input classes to Data. It's not a table in database!!! Move it right into controller. And make record to use less space.